Functions/FirebaseFunctions/Public/FirebaseFunctions/FIRFunctions.h

```diff
@@ -44,6 +44,15 @@ NS_SWIFT_NAME(Functions)
  */
 + (instancetype)functionsForRegion:(NSString *)region NS_SWIFT_NAME(functions(region:));
 
+/**
+ * Creates a Cloud Functions client with the default app and given custom domain.
+ * @param customDomain A custom domain for the http trigger, such as "https://mydomain.com".
+ */
+// clang-format off
+// because it incorrectly breaks this NS_SWIFT_NAME.
++ (instancetype)functionsForCustomDomain:(NSString *)customDomain NS_SWIFT_NAME(functions(customDomain:));
```

Comment:
Can developers provide both a region and custom domain, or are they mutually exclusive?

Reply:
They are mutually exclusive as `region` has no meaning when `customDomain` is set, the URL simply becomes `${customDomain}/${functionName}`